app/models/bulk_upload.rb

```diff
@@ -107,7 +107,7 @@ def initialize(zip_file = nil)
     end
 
     def temp_dir
-      @temp_dir ||= Dir.mktmpdir(nil, BULK_UPLOAD_ZIPFILE_DEFAULT_ROOT_DIRECTORY)
+      @temp_dir ||= Dir.mktmpdir(nil, Whitehall.bulk_upload_tmp_dir)
```

Comment:
Do we need call `Dir.mktmpdir`, if we're already creating the directory in the `Whitehall.bulk_upload_tmp_dir` function with `FileUtils.mkdir_p`?

Reply:
I.e. should we just have `Whitehall.bulk_upload_tmp_dir` return the path and call FileUtils.mkdir_p here?

Reply:
Ah righto! This makes much more sense now - we're passing a directory prefix, in which we make temporary directories within. Thanks for clarification!